frame/treasury/src/tests.rs

```diff
@@ -60,6 +60,8 @@ impl Contains<u64> for TenToFourteen {
 	fn sorted_members() -> Vec<u64> {
 		vec![10, 11, 12, 13, 14]
 	}
+	#[cfg(feature = "runtime-benchmarks")]
+	fn add(_: &u64) { unimplemented!() }
```

Comment:
Hmm?

Reply:
See https://github.com/paritytech/substrate/pull/5287/files#diff-0010d009b169768f89902f63e50e3c10R174

Maybe it can be done better? let me know